quartz/components/ContentMeta.tsx

```diff
@@ -23,22 +26,28 @@ export default ((opts?: Partial<ContentMetaOptions>) => {
     const text = fileData.text
 
     if (text) {
-      const segments: string[] = []
+      const segments: (string | JSX.Element)[] = [];
 
       if (fileData.dates) {
         segments.push(formatDate(getDate(cfg, fileData)!, cfg.locale))
       }
 
       // Display reading time if enabled
       if (options.showReadingTime) {
+        if(options.showComma){
+          segments.push(",")
+        }
         const { minutes, words: _words } = readingTime(text)
         const displayedTime = i18n(cfg.locale).components.contentMeta.readingTime({
           minutes: Math.ceil(minutes),
         })
         segments.push(displayedTime)
       }
-
-      return <p class={classNames(displayClass, "content-meta")}>{segments.join(", ")}</p>
+      // map each element to a span to allow for formatting
+      const segmentsElements = segments.map((segment, index) => (
+        <span key={index}>{index > 0 ? " " : ""}{segment}</span>
```

Comment:
this spacing should be done via CSS and not whitespace text

Reply:
Worth to just make this into a list then done all the formatting in css.